bindings/py/cpp_src/bindings/algorithms/py_Connections.cpp

```diff
@@ -124,7 +124,7 @@ R"(Returns numActiveConnectedSynapsesForSegment)");
                 [](void *dataPtr) {
                     delete reinterpret_cast<std::vector<SynapseIdx>*>(dataPtr); });
             // Call the C++ method.
-            self.computeActivity(*activeConnectedSynapses, *activePotentialSynapses,
+            *activeConnectedSynapses = self.computeActivity(*activePotentialSynapses,
```

Comment:
Same issue here.  The vector it's assigning to is allocated with a full buffer of zeros, which will get discarded w/o being used.